platforms/documentation/docs/src/docs/userguide/dep-man/02-declaring-dependency-versions/dynamic_versions.adoc

```diff
@@ -13,89 +13,91 @@
 // limitations under the License.
 
 [[sec:dynamic_versions_and_changing_modules]]
-= Handling versions which change over time
+= Handling dynamic versions
+
+There are many situations when you might need to use the latest version of a specific module dependency or the latest within a range of versions. This is often necessary during development or when creating a library that is compatible with various dependency versions.
 
-There are many situations when you want to use the latest version of a particular module dependency, or the latest in a range of versions.
-This can be a requirement during development, or you may be developing a library that is designed to work with a range of dependency versions.
 You can easily depend on these constantly changing dependencies by using a _dynamic version_.
-A <<dynamic_versions.adoc#sub:declaring_dependency_with_dynamic_version,dynamic version>> can be either a version range (e.g. `2.+`) or it can be a placeholder for the latest version available e.g. `latest.integration`.
 
-Alternatively, the module you request can change over time even for the same version, a so-called <<dynamic_versions.adoc#sub:declaring_dependency_with_changing_version,changing version>>.
-An example of this type of _changing module_ is a Maven `SNAPSHOT` module, which always points at the latest artifact published.
-In other words, a standard Maven snapshot is a module that is continually evolving, it is a "changing module".
+A <<dynamic_versions.adoc#sub:declaring_dependency_with_dynamic_version,dynamic version>> can be either a version range (e.g., `2.+`) or a placeholder for the latest version available, e.g., `latest.integration`.
```

Comment:
It would be nice if we can specifically call out dependency locking here. 

I would say in general, if you are using dynamic versions, you should _almost always_ use dependency locking. Without dependency locking, dynamic versions lead to non-reproducible builds -- which should be avoided if at all possible. 

Would we be able to add a quick note/warning here saying you should almost always use dependency locking along with dynamic versions?